src/reformat_libyuv.c

```diff
@@ -103,6 +103,8 @@ unsigned int avifLibYUVVersion(void)
 #define I420ToARGBMatrixFilter NULL
 #define I210ToARGBMatrixFilter NULL
 #define I010ToARGBMatrixFilter NULL
+#define I010AlphaToARGBMatrix NULL
```

Comment:
these two functions were also available since `LIBYUV_VERSION` 1780.

can you please move them to line 122 ? in the `LIBYUV_VERSION < 1780` section?

Reply:
Oops, would have been bad to break it for other versions.. Fixed.